.eslintrc.js

```diff
@@ -19,9 +19,21 @@ module.exports = {
       ImportDeclaration: { multiline: true, minProperties: 6 },
       ExportDeclaration: { multiline: true, minProperties: 6 },
     }],
+    'no-unused-expressions': 0,
```

Comment:
I know these options were added because our tests are using chai, but the problem with changing the configuration like this, is you're also changing/removing a rule for our javascript files under libs, which we want to keep.
To test this, add this silly code example to the load area method of the libs/utils/utils.js: 
```
const n = 1;
n + 1;
alert(n);
```
the `n + 1`  expression should cause eslint to throw an error, but you've disabled that now in the eslint settings.

To be honest my vote would almost go to having a separate eslint config for the test files. Then you can configure and change as much stuff as you want for the tests! :)
 https://eslint.org/docs/latest/user-guide/configuring/configuration-files

Reply:
I just checked that case and I cannot reproduce the behaviour you mention. If I add the sample code above in, for example, `libs/utils/utils.js`, `npm run lint` gives an error for the `n + 1` line. Though it's a `chai-friendly/no-unused-expressions`, and not anymore the generic `no-unused-expressions`.

Reply:
Hmm, that's weird. I just reran my lint and am getting the same result as you now. Maybe I forgot to run `npm ci` or something

Reply:
Well, that clears up my comment. Looks good to me!